web_src/css/modules/divider.css

```diff
@@ -0,0 +1,35 @@
+.ui.divider {
+  margin: 1rem 0;
+  height: 0;
+  font-weight: var(--font-weight-medium);
+  text-transform: uppercase;
+  color: var(--color-text);
+  font-size: 1rem;
+  width: 100%;
+}
+
+.ui.divider:not(.horizontal) {
+  border-top: 1px solid var(--color-secondary);
+}
+
+.ui.horizontal.divider {
+  display: flex;
+  align-items: center;
+  padding: 7px 0;
+}
+
+.ui.horizontal.divider::before,
+.ui.horizontal.divider::after {
+  content: "";
+  height: 1px;
```

Comment:
I found this `height` has no effect because of the `border-top`? so just wondering if the height can be removed.

Reply:
Right. IIRC, this was from an earlier version that used background of 1px. Need to check which method is better.

Reply:
Removed it, still looks ok in terms of centering.